Documentation/doc/scripts/html_output_post_processing.py

```diff
@@ -70,7 +70,7 @@ def clean_doc():
     duplicate_files=list(package_glob('./*/jquery.js'))
     duplicate_files.extend(package_glob('./*/dynsections.js'))
     duplicate_files.extend(package_glob('./*/resize.js'))
-    duplicate_files.extend(package_glob('./*/stylesheet.css'))
+    duplicate_files.extend(package_glob('./*/cgal_stylesheet.css'))
```

Comment:
```suggestion
    duplicate_files.extend(package_glob('./*/stylesheet.css'))
    duplicate_files.extend(package_glob('./*/cgal_stylesheet.css'))
```

Reply:
Why as `stylesheet.css` does not exist anymore (as far as I can see in the 1.8.13 ... versions)?

Reply:
You are right. Your pull-request is already fine with recent versions of CGAL. I searched in an older version I had on my computer.